algorithms/java/DFS.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.*;
+
+class Graph
+{
+    private int V; 
+
+    private LinkedList<Integer> adj[];
+
+
+    @SuppressWarnings("unchecked")
+    Graph(int v)
+    {
+        V = v;
+        adj = new LinkedList[v];
+        for (int i=0; i<v; ++i)
+            adj[i] = new LinkedList();
+    }
+
+
+    void addEdge(int v, int w)
+    {
+        adj[v].add(w);
+    }
+
+
+    void DFSUtil(int v,boolean visited[])
+    {
+        /
+        visited[v] = true;
+        System.out.print(v+" ");
```

Comment:
please remove this slash from the code it causes an error while compiling.
Thanks